tasks/packer/src/main/java/com/walmartlabs/concord/plugins/packer/PackerTask.java

```diff
@@ -0,0 +1,43 @@
+package com.walmartlabs.concord.plugins.packer;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2018 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+import java.util.Map;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import ca.vanzyl.concord.plugins.tool.ToolCommand;
+import ca.vanzyl.concord.plugins.tool.ToolInitializer;
+import ca.vanzyl.concord.plugins.tool.ToolTaskSupport;
+import com.walmartlabs.concord.sdk.InjectVariable;
+
+@Named("packer")
+public class PackerTask extends ToolTaskSupport
+{
+    @InjectVariable("defaults")
+    private Map<String, Object> defaults;
```

Comment:
Typically in v1 each task requests their defaults using a task-specific defaults, e.g. `packerDefaults`, so that they can be separated in the [default variables file](https://github.com/walmartlabs/concord/blob/master/server/dist/src/main/resources/concord-server.conf#L98) or in [the default process configuration policy](https://concord.walmartlabs.com/docs/getting-started/policies.html#default-process-configuration-rule).

Reply:
So if that is changed to `packerDefaults`, what does the stanza in the `server.conf` look like? Asking so I can add it to the documentation.

Reply:
Something like this:
```
# server.conf

concord-server {
  process {
    defaultConfiguration = "/path/to/default_vars.yml"
  }
}
```

```yaml
# default_vars.yml
configuration:
  arguments:
    packerDefaults:
      xyz: 123
```

Or the same thing using [policies](https://concord.walmartlabs.com/docs/getting-started/policies.html#default-process-configuration-rule).

Ultimately, all those values end up as regular flow variables.